text.cpp

```diff
@@ -91,7 +91,16 @@ class Text {
       }
 
       auto isMatchingSymbol = [&](const Symbol& symbol) -> bool {
-        return symbol.id == static_cast<uint32_t>(c);
+        /* c is char_t here, so ...
```

Comment:
forgot about this one? @doTulio 
It's a nit, so if you don't change it I'll just merge anyway in about 10 minutes or so.